diff --git a/node_modules/nepali-date-converter/dist/lib/date-config.js b/node_modules/nepali-date-converter/dist/lib/date-config.js
--- a/node_modules/nepali-date-converter/dist/lib/date-config.js
+++ b/node_modules/nepali-date-converter/dist/lib/date-config.js
@@ -1180,7 +1180,7 @@ exports.dateConfigMap = {
     },
     '2084': {
         Baisakh: 31,
-        Jestha: 32,
+        Jestha: 31,
         Asar: 31,
         Shrawan: 32,
         Bhadra: 31,
@@ -1193,7 +1193,7 @@ exports.dateConfigMap = {
         Poush: 29,
         Magh: 29,
         Falgun: 30,
-        Chaitra: 31,
+        Chaitra: 30,
     },
     '2085': {
-        Baisakh: 30,
+        Baisakh: 31,
         Jestha: 32,
         Asar: 31,
@@ -1206,7 +1206,7 @@ exports.dateConfigMap = {
         Poush: 29,
         Magh: 30,
         Falgun: 29,
-        Chaitra: 31,
+        Chaitra: 30,
     },
     '2086': {
         Baisakh: 31,
diff --git a/node_modules/nepali-date-converter/dist/nepali-date-converter.es5.js b/node_modules/nepali-date-converter/dist/nepali-date-converter.es5.js
--- a/node_modules/nepali-date-converter/dist/nepali-date-converter.es5.js
+++ b/node_modules/nepali-date-converter/dist/nepali-date-converter.es5.js
@@ -1203,7 +1203,7 @@ var dateConfigMap = {
     },
     '2084': {
         Baisakh: 31,
-        Jestha: 32,
+        Jestha: 31,
         Asar: 31,
         Shrawan: 32,
         Bhadra: 31,
@@ -1216,7 +1216,7 @@ var dateConfigMap = {
         Poush: 29,
         Magh: 29,
         Falgun: 30,
-        Chaitra: 31,
+        Chaitra: 30,
     },
     '2085': {
-        Baisakh: 30,
+        Baisakh: 31,
         Jestha: 32,
         Asar: 31,
@@ -1229,7 +1229,7 @@ var dateConfigMap = {
         Poush: 29,
         Magh: 30,
         Falgun: 29,
-        Chaitra: 31,
+        Chaitra: 30,
     },
     '2086': {
         Baisakh: 31,
diff --git a/node_modules/nepali-date-converter/dist/nepali-date-converter.umd.js b/node_modules/nepali-date-converter/dist/nepali-date-converter.umd.js
--- a/node_modules/nepali-date-converter/dist/nepali-date-converter.umd.js
+++ b/node_modules/nepali-date-converter/dist/nepali-date-converter.umd.js
@@ -1210,7 +1210,7 @@
         '2084': {
             Baisakh: 31,
-            Jestha: 32,
+            Jestha: 31,
             Asar: 31,
             Shrawan: 32,
             Bhadra: 31,
@@ -1222,7 +1222,7 @@
             Poush: 29,
             Magh: 29,
             Falgun: 30,
-            Chaitra: 31,
+            Chaitra: 30,
         },
         '2085': {
-            Baisakh: 30,
+            Baisakh: 31,
             Jestha: 32,
             Asar: 31,
@@ -1235,7 +1235,7 @@
             Poush: 29,
             Magh: 30,
             Falgun: 29,
-            Chaitra: 31,
+            Chaitra: 30,
         },
         '2086': {
             Baisakh: 31,
